src/parkapi_sources/converters/opendata_swiss/converter.py

```diff
@@ -0,0 +1,79 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from abc import ABC
+
+import requests
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator
+
+from parkapi_sources.converters.base_converter.pull import GeojsonInput, PullConverter
+from parkapi_sources.exceptions import ImportParkingSiteException, ImportSourceException
+from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
+
+from .models import OpenDataSwissFeatureInput
+
+
+class OpenDataSwissBasePullConverter(PullConverter, ABC):
```

Comment:
Why do you create a Base class for just one Child class? Do we expect more Child classes?